.github/workflows/main.yml

```diff
@@ -23,7 +22,8 @@ jobs:
           configVdf: ${{ secrets.STEAM_CONFIG_VDF}}
           ssfnFileName: ${{ secrets.STEAM_SSFN_FILE_NAME }}
           ssfnFileContents: ${{ secrets.STEAM_SSFN_FILE_CONTENTS }}
-          appId: 1234560
+          appId: ${{ secrets.TEST_APP_ID }}
+          firstDepotIdOverride: ${{ secrets.TEST_FIRST_DEPOT_ID_OVERRIDE }}
```

Comment:
This would extend the public API of this action. 

Is there no way to just use the recommended approach from steam themselves?

Reply:
I'm not familiar with this. What recommended approach are you referring to?

Reply:
With public API I simply mean what you configure the action with:

```
  with:
    parameter: value
```

In general, as part of GameCI Actions philosophy we try to limit the amount of parameters that the end-user would have to understand to a minimum. We always try to deduce information internally inside the action where possible.

Perhaps it's not possible to do that in this case, but I don't have sufficient context to judge this. This is why I'm asking as a question.

Reply:
Ah sorry I understand the aversion to modifying the public API - my question was more about this part:

> Is there no way to just use the recommended approach from steam themselves?

I'm not sure what recommended approach from Steam you're referring to here

Reply:
In terms of context, what I've learned in this exercise is that a certain configuration of DLC in Steamworks results in a first depot whose ID is not `appId + 1`, but rather is exactly equal to `appId`, so the existing functionality wouldn't have worked for that 😢 

Even thought this DLC was just made for testing, it's highlighted a situation that real users can face if they're using Game.CI to build/deploy some DLC.

So I added this as a backwards-compatible option for handling this DLC case, and possibly other cases too that we're still not aware of.